ultraLatency.py

```diff
@@ -36,7 +36,7 @@
 parser.add_option("--fakeTTC", action="store_true", dest="fakeTTC",
                   help="Set up for using AMC13 local TTC generator", metavar="fakeTTC")
 
-parser.set_defaults(scanmin=153,scanmax=172,nevts=500)
+parser.set_defaults(scanmin=153,scanmax=172,nevts=500, amc13local=False, t3trig=False,fakeTTC=false,internal=False)
```

Comment:
`store_true` options default to `False` (or `None` which evaluates to `False` if checked in an `if` statement), so this is unnecessary